scss/grid/_grid.scss

```diff
@@ -116,12 +116,14 @@ th.column {
   }
 
   //Collapsed logic
-  .collapse {
-    > tbody > tr > td.large-#{$i},
-    > tbody > tr > th.large-#{$i} {
-      padding-right: 0;
-      padding-left: 0;
-      width: -zf-grid-calc-px($i, $grid-column-count, $global-width) + $global-gutter;
+  .collapse > tbody > tr {
```

Comment:
Why the change here?

Reply:
To reduce repetition; instead of:

```scss
 .collapse {
     > tbody > tr > td.large-#{$i},
     > tbody > tr > th.large-#{$i} {
```

the selector can just be:

```scss
 .collapse > tbody > tr {
     > td.large-#{$i},
     > th.large-#{$i} {
```

This way we're only typing the `tbody > tr` once.